### PR TITLE
[DOCS] Moved some more tutorial files around

### PR DESCRIPTION
**Pull request by spbail**
Changes proposed in this pull request:
This is a little bit of a hodge-podge of things, with the main goal to restructure the tutorials section a little more as we discussed earlier:
- Moved "quickly explore expectations" section under Tutorials as discussed last week
- Moved "how to create expectations" under Tutorials, as a deep dive once you've established a context. Tbh it contains a lot of duplicate content to "getting started" - I think it might make sense to merge the two eventually, but I'm keeping it for now.
- Moved contributing out of tutorials so we don't have to maintain two landing pages for it, but kept the link on the tutorials landing page
- Some minor cleanups for references and broken images where I found them
- @abegong We discussed moving the "customizing your deployment" section into its own top-level tutorial, but the way it's written is actually more fitting as an optional last page in the getting started tutorial. It's more of a link list of pointers. I kept it in the original place and just renamed it to "Optional: "

The next step is to actually change the "getting started" tutorial content, which I've been putting off until we have an idea of how things fit together.

### CODE REVIEW

**Review by jcampbell — APPROVED** [review]
Woot!

File: docs/contributing.rst [hunk 1]
```diff
@@ -1,7 +1,98 @@
...
+      </span> Identify the type of contribution that you want to make</h2>
+   </embed>
+
+.. container:: custom-indented-paragraph
+
+   Issues in GitHub are a great place to start. Check out the `help wanted <https://github.com/great-expectations/great_expectations/labels/help%20wanted>`__ and `good first issue <https://github.com/great-expectations/great_expectations/labels/good%20first%20issue>`__ labels. Comment to let everyone know you’re working on it.
+
+   If there’s no issue for what you want to work on, please create one. Add a comment to let everyone know that you're working on it. We prefer small, incremental commits, because it makes the thought process behind changes easier to review.
+
+  Our :ref:`levels_of_maturity` grid provides guidelines for how the maintainers of Great Expectations evaluate levels of maturity of a feature.
```
Comment (abegong) [inline docs/contributing.rst]: Indentation and link are broken...

File: docs/guides/tutorials.rst [hunk 2]
```diff
@@ -5,20 +5,23 @@
 Tutorials
 #########
 
-These tutorials will teach you everything you need to know to get up and running with Great Expectations.
+These tutorials will teach you the basics of what you need to know to get up and running with Great Expectations.
```
Comment (abegong) [inline docs/guides/tutorials.rst:8]: I ❤️ ❤️ ❤️  all the revisions on this page

File: docs/guides/tutorials.rst [hunk 3]
```diff
@@ -5,20 +5,23 @@
...
+* If you're a **new user** of Great Expectations, check out :ref:`tutorials__getting_started` to learn important concepts by setting up your first local deployment of Great Expectations. **We recommend you get started here!**
+
+* If you already have a working deployment of Great Expectations and want a deep dive into how to create and edit Expectations, go to the :ref:`tutorials__create_expectations` tutorial.
+
+* Finally, if you're interested in contributing to the open source project, thank you! Please check out :ref:`the Contributing section <contributing>` to learn how.
 
 
 .. toctree::
-   :maxdepth: 2
+   :hidden:
```
Comment (abegong) [inline docs/guides/tutorials.rst:22]: Well, that's a handy directive to know...

File: docs/guides/tutorials/getting_started/customize_your_deployment.rst [hunk 4]
```diff
@@ -1,9 +1,9 @@
 .. _tutorials__getting_started__customize_your_deployment:
 
-Customize your deployment
+Optional: Customize your deployment
```
Comment (abegong) [inline docs/guides/tutorials/getting_started/customize_your_deployment.rst:3]: I thought we were going to pull this out into its own top-level tutorial.

Is this an intermediate step towards that end?

**Review by abegong — COMMENTED** [review]
LGTM.

There are a couple small formatting issues to fix before merging.

File: docs/guides/tutorials.rst (hunk 3 above)
Comment (spbail) [reply docs/guides/tutorials.rst:22]: It took me a while to understand why we have these awkwardly placed TOCs at the bottom of each page :) This way you can still define the child tree structure but the TOC isn't visible if it's not needed. I figured we could slowly overhaul other pages like this as we encounter them.

File: docs/guides/tutorials/getting_started/customize_your_deployment.rst (hunk 4 above)
Comment (spbail) [reply docs/guides/tutorials/getting_started/customize_your_deployment.rst:3]: See my comment in the initial commit message - the section is very much not written as a tutorial and really just contains a few links, so it just made more sense to keep it as an optional "where to go from here".

File: docs/contributing.rst (hunk 1 above)
Comment (spbail) [reply docs/contributing.rst]: Fixing...

Comment (spbail) [reply docs/contributing.rst]: Fixed in the latest commit

File: docs/guides/tutorials/getting_started/customize_your_deployment.rst (hunk 4 above)
Comment (spbail) [reply docs/guides/tutorials/getting_started/customize_your_deployment.rst:3]: Oh and regarding this being an intermediate step I'd say we could think of making this a full-on tutorial, but I didn't realize how far it was from being self-standing as a tutorial, so not for now.

File: docs/guides/tutorials.rst (hunk 3 above)
Comment (abegong) [reply docs/guides/tutorials.rst:22]: ❤️

File: docs/guides/tutorials/getting_started/customize_your_deployment.rst (hunk 4 above)
Comment (abegong) [reply docs/guides/tutorials/getting_started/customize_your_deployment.rst:3]: 🤦 for not reading your original commit message.

Yes, this makes sense.

**Review by abegong — APPROVED** [review]
:100:

**Pull request merged**